src/create-config.ts

```diff
@@ -0,0 +1,322 @@
+import { ethers } from 'ethers';
+import { Choice, PromptObject } from 'prompts';
+import { encode } from '@api3/airnode-abi';
+import { AirnodeRrpAddresses, RequesterAuthorizerWithAirnodeAddresses } from '@api3/airnode-protocol';
+import { deriveEndpointId } from '@api3/airnode-admin';
+import { ChainConfig, NodeSettings, Triggers, Trigger, ApiCredentials, Config } from '@api3/airnode-node';
+import {
+  AirkeeperConfig,
+  AirkeeperChainConfig,
+  Triggers as AirkeeperTriggers,
+  Subscriptions,
+  Templates as AirkeeperTemplates,
+  Endpoints as AirkeeperEndpoints,
+} from '@api3/airkeeper/dist/src/validator';
+import { OperationsRepository } from './types';
+import { promptQuestions, readOperationsRepository, writeOperationsRepository } from './utils/filesystem';
+import { runAndHandleErrors } from './utils/cli';
+import { chainNameToChainId, DapiServerInterface } from './utils/evm';
+
+const questions = (choices: Choice[]): PromptObject[] => {
+  return [
+    {
+      type: 'autocomplete',
+      name: 'apiName',
+      message: 'What is the name of the API Integration?',
+      choices: choices,
+    },
+    {
+      type: 'confirm',
+      name: 'airnodeAuthorizer',
+      message: 'Do you want to include the airnode authorizer?',
+      initial: false,
+    },
+    {
+      type: 'confirm',
+      name: 'airnodeHeartbeat',
+      message: 'Do you want to enable the airnode heartbeat?',
+      initial: false,
+    },
+  ];
+};
+
+const main = async () => {
+  const operationsRepository = readOperationsRepository();
+  const apiChoices = Object.keys(operationsRepository.apis).map((api) => ({ title: api, value: api })) as Choice[];
+  const response = await promptQuestions(questions(apiChoices));
+  const apiData = operationsRepository.apis[response.apiName];
+  const secrets = [];
+
+  //// Build Config.json  ////
+
+  // Get all the chains the API will be deployed on
+  const apiChains = [
+    ...new Set(
+      Object.keys(apiData.beacons).flatMap((beacon) => apiData.beacons[beacon].chains.map((chain) => chain.name))
+    ),
+  ];
+
+  const chains: ChainConfig[] = apiChains.map((chainName) => {
```

Comment:
```suggestion
  const chains = apiChains.map((chainName) => {
```
As above, this type isn't needed.